styleguide/source/_patterns/03-templates/01-home/_01-home.scss

```diff
@@ -1,8 +1,18 @@
 .layout-home {
+  padding: 0 20px;
 
   @include breakpoint($bp-med-home min-width) {
     padding: 0 30px;
   }
+
+  .col-width-3,
```

Comment:
@rpama I'm not sure these changes were necessary, but I'm fine with merging them and then removing them later once #201 is merged.

Reply:
So, if I recall correctly from Friday, the news_item stuff was getting some extra spacing on the home page and it bugged me so I decided this was a the easiest way to fix it (although not entirely elegant).